### ISSUE TITLE
The 'Update Settings' button under 'My Covid Alerts' page is not opening up my phone settings.

### ISSUE THREAD

**Issue by mboskin** [covid-alert-ny/covid-green-app#126]
Android | Galaxy S6+ | 8/27/2020 5:40PM | Raghunath Gajjala

The 'Update Settings' button under 'My Covid Alerts' page (under App Settings menu) is not opening up my phone settings.

**Comment by floridemai** [comment]
@mboskin I think this was happening on older versions. Do you know the version this was tested against?

**Comment by mboskin** [comment]
Linh, please confirm the following.

Android | Galaxy S6+ | 8/27/2020 5:40PM | Raghunath Gajjala

Yesterday at 5:40 PM they were testing the current build.

_____________________

www.nystec.com<http://www.nystec.com/>

**Comment by mboskin** [comment]
Mike, you can cut the loop of comm, just ask Raghu to confirm in the Teams chat for testing
thx

From: Michael Boskin <mboskin@nystec.com>
Sent: Friday, August 28, 2020 11:55 AM
To: project-vagabond/covid-green-app <reply@reply.github.com>; project-vagabond/covid-green-app <covid-green-app@noreply.github.com>; Le, Linh (ITS) <Linh.Le@its.ny.gov>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [project-vagabond/covid-green-app] The 'Update Settings' button under 'My Covid Alerts' page is not opening up my phone settings. (#126)


ATTENTION: This email came from an external source. Do not open attachments or click on links from unknown senders or unexpected emails.

Linh, please confirm the following.

Android | Galaxy S6+ | 8/27/2020 5:40PM | Raghunath Gajjala

Yesterday at 5:40 PM they were testing the current build.

_____________________
Michael Boskin
Senior Consultant, NYSTEC
(518) 496-7667 – Mobile
mboskin@nystec.com<mailto:mboskin@nystec.com>

www.nystec.com<https://protect2.fireeye.com/v1/url?k=da29bc60-860f845e-da2b4555-000babda0031-2c8c568f2ac6fe30&q=1&e=8cc77045-f4a0-4780-b041-ff3a199e6bec&u=http%3A%2F%2Fwww.nystec.com%2F>

**Comment by floridemai** [comment]
@mboskin A couple of updates on the ticket:
✔️  I have update the page to be in sync with the other closeness sensing screen. The existing code was old and even if in essence could have worked, it wasn't up to date.
✔️  Updated text to match designs
❗ The reported bug does indeed exist but a couple of notes should be made. Only happens on Android. Only happens when Closeness is active, you tap the button and then in the settings screen you don't do anything and instead of using the back arrow or phone back you choose to "Open app" from that screen which re-opens the app. Furthermore if you close the app and then re-open it, the button is again working properly.

My current changes should be in in the next release, prior to Monday's build.

@mboskin Fixing this could take quite a bit of time as it involves looking into android native code when the app starts. Not sure if we should put it on the top of our list. cc @dharding @arybka777 @kevin-sheldon. For the time being, this has been looked into and addressed and I'll re-assign it.

Note  to developers:
We open ENS settings with
```
        await IntentLauncher.startActivityAsync(
          'com.google.android.gms.settings.EXPOSURE_NOTIFICATION_SETTINGS'
        );
```
and subsequent calls error with
```
[Error: IntentLauncher activity is already started. You need to wait for its result before starting another activity.]
```
 for obvious reasons.

**Comment by colmharte** [comment]
Changing this low priority, its an edge case and there are simple work arounds
